Purchases.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  Purchases.swift
+//  Purchases
+//
+//  Created by Andrés Boedo on 8/24/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
```

Comment:
this file is needed so that Xcode can link swift stuff correctly for the static library dependency. Actually, I'll write that as a comment in the file in case anyone wonders in the future.